app/Main.hs

```diff
@@ -30,21 +44,34 @@ main = execParser opts >>= dotEnv
      <> header "dotenv - loads options from dotenv files" )
 
 config :: Parser Options
-config = Options
-     <$> some (strOption (
-                  long "file"
-                  <> short 'f'
-                  <> metavar "FILE"
-                  <> help "File to read for options" ))
+config =
+  Options
+     <$> argument str (metavar "PROGRAM")
+
+     <*> strOption
+          ( long "env"
+          <> value ".env"
+          <> metavar "\"DOTENV FILE\""
+          <> help "File with the env variables" )
 
-     <*> argument str (metavar "PROGRAM")
+     <*> strOption
+          ( long "env-example"
+          <> value ".env.example"
+          <> metavar "\"DOTENV EXAMPLE\" FILE"
+          <> help "File with all the necesary env variables" )
 
-     <*> switch ( long "overload"
-                  <> short 'o'
-                  <> help "Specify this flag to override existing variables" )
+     <*> switch
+           ( long "overload"
```

Comment:
It would be good to be consistent in wording: is it "overload" or "override"? If it's overload, then we could edit name of record selector in `Options` to match.